hi-ml-histopathology/src/histopathology/utils/naming.py

```diff
@@ -32,6 +32,10 @@ class TileKey(str, Enum):
     SPLIT = 'split'
     TILE_X = 'tile_x'
     TILE_Y = 'tile_y'
+    TILE_LEFT = 'left'
```

Comment:
Do you envision us keeping the `TILE_X/Y` keys in addition to these new ones? Or should we refactor them out of existence later on?

Reply:
As @vale-salvatelli, I think we should deprecate them at some point.